heron/packing/src/java/com/twitter/heron/packing/roundrobin/RoundRobinPacking.java

```diff
@@ -343,4 +351,37 @@ private void validatePackingPlan(PackingPlan plan) throws PackingException {
       }
     }
   }
+
+  /*
+   * read the current packing plan with update parallelism to calculate a new packing plan
```

Comment:
is "current packing plan" accurate?

Reply:
this packing plan is the latest from zk.